### PR TITLE
Improve layout of receive/sending lightning/sending on-chain screens

### LINKED ISSUE #1484: Enhancement: Payment successful screen often doesn't fit on screen
### Describe the bug

There are lots of different screen resolutions of course, but I think it is possible to make the "Payment successful" screen so it usually fits on a phone screen without the need to scroll. Right now I often have to scroll, although my phone is 1440 x 3200 px.

### Reproduce

-

### Zeus version

v0.7.6

### Node interface

LND (REST)

### Network

Tor

### Device

Samsung Galaxy S20+

### Device operating system

Android 13

### Log output

_No response_

### PR DESCRIPTION
**Pull request by myxmaster**
# Description

This fixes #1484.

grafik
grafik
grafik

This pull request is categorized as a:

- [ ] New feature
- [ ] Bug fix
- [ ] Code refactor
- [ ] Configuration change
- [ ] Locales update
- [ ] Quality assurance 
- [x] Other

## Checklist
- [x] I’ve run `yarn run tsc` and made sure my code compiles correctly
- [x] I’ve run `yarn run lint` and made sure my code didn’t contain any problematic patterns
- [x] I’ve run `yarn run prettier` and made sure my code is formatted correctly
- [x] I’ve run `yarn run test` and made sure all of the tests pass

## Testing

If you modified or added a utility file, did you add new unit tests?

- [ ] No, I’m a fool
- [ ] Yes
- [x] N/A

I have tested this PR on the following platforms (please specify OS version and phone model/VM):

- [x] Android
- [ ] iOS

I have tested this PR with the following types of nodes (please specify node version and API version where appropriate):

- [x] LND (REST)
- [ ] LND (Lightning Node Connect)
- [ ] Core Lightning (c-lightning-REST)
- [ ] Core Lightning (Spark)
- [ ] Eclair
- [x] LndHub

### Locales
- [x] I’ve added new locale text that requires translations
- [x] I’m aware that new translations should be made on the Zeus [Transfix page](https://app.transifex.com/ZeusLN/zeus/) and not directly to this repo

### Third Party Dependencies and Packages

- [ ] Contributors will need to run `yarn` after this PR is merged in
- [ ] 3rd party dependencies have been modified:
    * verify that `package.json` and `yarn.lock` have been properly updated
    * verify that dependencies are installed for both iOS and Android platforms

### Other:

- [ ] Changes were made that require an update to the README
- [ ] Changes were made that require an update to onboarding

### CODE REVIEW

File: components/PaidIndicator.tsx [hunk 1]
```diff
@@ -13,15 +13,15 @@ interface PaidIndicatorProps {
...
+function PaidIndicator(props: PaidIndicatorProps) {
     const { size } = props;
     const randomNumber = Math.ceil(Math.random() * 3) - 1;
 
     return (
         <View
             style={{
                 position: 'absolute',
-                alignSelf: 'center',
+                top: -220,
```
Comment (kaloudis) [inline components/PaidIndicator.tsx:24]: not crazy about fixes that include adding negative margins

**Comment by kaloudis** [comment]
What do these pages look like at higher resolution, say on a tablet?

File: components/PaidIndicator.tsx (hunk 1 above)
Comment (myxmaster) [reply components/PaidIndicator.tsx]: Improved this in latest commit.

**Comment by myxmaster** [comment]
I checked it on a 10" tablet and it didn't look good, so I fixed the size to make it look the same on all devices.

Tablet screenshots:

**Comment by kaloudis** [comment]
I don't think this is enough of an improvement. I'd like to do a refactor where the action buttons hang to the bottom of the screen and the logo and animations are scaled and spaced out depending on the size of the screen

**Comment by myxmaster** [comment]
I do like the idea, but then it would be inconsistent.

grafik

**Comment by kaloudis** [comment]
> I do like the idea, but then it would be inconsistent.
> 
> grafik

Let's make it consistent then!

**Pull request merged**